src/Summary.js

```diff
@@ -1,10 +1,37 @@
 import { useTasksCount } from "./api/api.hooks";
+import { useTasksQuery } from "./api/api.hooks";
 
 export const Summary = () => {
-  const { data: tasksCount } = useTasksCount();
+  const { data: tasks } = useTasksQuery();
+  var taskEffort = 0;
+  var taskOpen = 0;
+  var effortOpen = 0;
+  if (tasks && tasks.length) {
+    for (var i = 0; i < tasks.length; i++) {
+      var task = tasks[i];
+      taskEffort = taskEffort + (task ? parseInt(tasks[i].effort || 0) : 0);
+      if (!task.archived) {
+        taskOpen = taskOpen + 1;
+        effortOpen = effortOpen + (parseInt(task.effort) || 0);
+      }
+    }
+  }
```

Comment:
This is a great example of place where we can use array transforms, specifically [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) and [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce), to make this code simpler, easier to read, and easier to maintain:

```suggestion
const {data} = useTasksQuery();
// fallback to an empty array if the tasks response is empty
const tasks = data || []; 

const tasksOpen = tasks.filter(task => !task.archived);
const totalEffort = tasks.reduce((acc, task) => acc + (parseInt(task.effort, 10) || 0), 0);
const openTotalEffort = tasksOpen.reduce((acc, task) => acc + (parseInt(task.effort, 10) || 0), 0);
```

Your count of `totalOpenTasks` will then just be `tasksOpen.length`.